test/python/sdpa/frost/test_sdpa_fwd_heuristics.py

```
@@ -0,0 +1,230 @@
+
+    s = torch.einsum("bhqd,bhkd->bhqk", q_gpu.float(), k_gpu.float()) / math.sqrt(D)
+    i = torch.arange(SQ, device="cuda").view(SQ, 1)
+    j = torch.arange(SKV, device="cuda").view(1, SKV)
+    s = s.masked_fill(j > i, float("-inf"))
+    torch.testing.assert_close(o_gpu, torch.einsum("bhqk,bhkd->bhqd", torch.softmax(s, dim=-1), v_gpu.float()).half(), atol=5e-2, rtol=3e-2)
+    torch.testing.assert_close(st_gpu.squeeze(-1), torch.logsumexp(s, dim=-1), atol=2e-3, rtol=2e-3)
+
+    # Autotune replay: the split entry round-trips through (engine_id, knobs).
+    eng_id, knobs = g.get_engine_and_knobs_at_index(split_idx)
+    assert knobs.split_kv == 4
+    g2, handles2, _ = _build_decodeish_graph()
```

Ruff (0.16.1)

[warning] Unpacked variable `handles2` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)